src/screens/menu/views/TurnAppBackOn.tsx

```diff
@@ -8,7 +8,7 @@ export const TurnAppBackOn = () => {
   const startExposureNotificationService = useStartExposureNotificationService();
 
   const onStart = useCallback(async () => {
-    await startExposureNotificationService();
+    await startExposureNotificationService(true);
```

Comment:
@smcmurtry You did not leave any comment here. Should I also set it to `false` ?

Reply:
I just checked and it looks like the `TurnAppBackOn` component isn't used anywhere. So it doesn't matter - this whole file could be deleted.

Reply:
s/could/should

😀

Reply:
Haha! I was in the process of doing it :)